include/Application/VggSdk.hpp

```diff
@@ -42,29 +42,29 @@ class VggSdk
 
   virtual ~VggSdk() = default;
 
+  // env
+  std::string getEnvKey();
+  void        setContainerKey(const std::string& containerKey);
+  void        setInstanceKey(const std::string& instanceKey);
+  void        setListenerKey(const std::string& listenerKey);
+
   // design document
-  const std::string designDocument(IndexType index = main_or_editor_daruma_index);
-  void designDocumentAddAt(const std::string& jsonPointer,
-                           const std::string& value,
-                           IndexType index = main_or_editor_daruma_index);
-  void designDocumentReplaceAt(const std::string& jsonPointer,
-                               const std::string& value,
-                               IndexType index = main_or_editor_daruma_index);
-  void designDocumentDeleteAt(const std::string& jsonPointer,
-                              IndexType index = main_or_editor_daruma_index);
+  const std::string designDocument();
+  void              designDocumentAddAt(const std::string& jsonPointer, const std::string& value);
```

Comment:
Please insert an empty line above to bypass clang-format's bug, because the formatting is not correct with the following two lines.

Reply:
Updated